<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>srhpreUI</class>
 <widget class="QDialog" name="srhpreUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>685</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>923</width>
    <height>685</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>923</width>
    <height>685</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QGroupBox" name="groupBox_11">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>540</y>
     <width>241</width>
     <height>131</height>
    </rect>
   </property>
   <property name="title">
    <string>Special Modeling Options</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>11</x>
      <y>21</y>
      <width>226</width>
      <height>99</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout_3">
     <item row="0" column="0">
      <widget class="QRadioButton" name="rbtnSpecialNone">
       <property name="text">
        <string>None</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QRadioButton" name="rbtnSpecialMomentumless">
       <property name="text">
        <string>Momentumless Water Sink/Source</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QRadioButton" name="rbtnSpecialInfiltration">
       <property name="text">
        <string>Infiltration</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QRadioButton" name="rbtnSpecialSediment">
       <property name="text">
        <string>Sediment Dropping into River</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>20</y>
     <width>261</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Case Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditCaseName"/>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>_SIF.DAT</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_5">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>201</width>
     <height>111</height>
    </rect>
   </property>
   <property name="title">
    <string>Time Setup</string>
   </property>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>171</width>
      <height>82</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Tstart (hr)</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="lineEditTStart">
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="lineEditTStep"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Total Time (hr)</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="lineEditTTotal"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Time Step (sec)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>50</y>
     <width>551</width>
     <height>24</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>Description</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditDescription"/>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="groupBox_4">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>250</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string>Simulation</string>
   </property>
   <widget class="QRadioButton" name="rbtnSimSteady">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>83</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Steady</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtnSimUnsteady">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>83</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Unsteady</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string>Solver</string>
   </property>
   <widget class="QRadioButton" name="rbtnSolverFlow">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>83</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Flow</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtnSolverMobile">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>83</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mobile</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string>Output Format</string>
   </property>
   <widget class="QRadioButton" name="rbtnOutputSMS">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>83</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SMS</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtnOutputTecplot">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>83</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tecplot</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>490</y>
     <width>201</width>
     <height>41</height>
    </rect>
   </property>
   <property name="title">
    <string>Output Unit</string>
   </property>
   <widget class="QRadioButton" name="rbtnOutputUnitSI">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>SI</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbtnOutputUnitEN">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>EN</string>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>190</y>
     <width>621</width>
     <height>431</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tabFlowSetting">
    <attribute name="title">
     <string>Flow Setting</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>191</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>Turbulent Model</string>
     </property>
     <widget class="QRadioButton" name="rbtnTurbPARA">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>83</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>PARA</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnTurbKE">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>83</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>KE</string>
      </property>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>321</width>
       <height>71</height>
      </rect>
     </property>
     <property name="title">
      <string>Mesh Unit</string>
     </property>
     <widget class="QWidget" name="layoutWidget_5">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>311</width>
        <height>47</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout_2">
       <item row="0" column="0">
        <widget class="QRadioButton" name="rbtnMUnitMETER">
         <property name="text">
          <string>METER</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QRadioButton" name="rbtnMUnitmm">
         <property name="text">
          <string>mm</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QRadioButton" name="rbtnMUnitKM">
         <property name="text">
          <string>KM</string>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QRadioButton" name="rbtnMUnitGScale">
         <property name="text">
          <string>GScale</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="rbtnMUnitFOOT">
         <property name="text">
          <string>FOOT</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QRadioButton" name="rbtnMUnitinch">
         <property name="text">
          <string>inch</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QRadioButton" name="rbtnMUnitMile">
         <property name="text">
          <string>MILE</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QGroupBox" name="groupBox_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>Manning's Setup</string>
     </property>
     <widget class="QRadioButton" name="rbtnManningConstant">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>83</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Constant</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnManningMaterial">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>20</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Material Type</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnManningDistributed">
      <property name="enabled">
       <bool>false</bool>
      </property>
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>101</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Distributed</string>
      </property>
     </widget>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>10</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Set parameter for Turbulence Model:</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="turbParaInput">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>50</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>30</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>(from 0.05 ~ 1.0)</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="mannTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>591</width>
       <height>141</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="mannLabel">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>160</y>
       <width>261</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>Flow Initial Cond.</string>
    </attribute>
    <widget class="QGroupBox" name="groupBox_12">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>311</width>
       <height>41</height>
      </rect>
     </property>
     <property name="title">
      <string>Initial Condition</string>
     </property>
     <widget class="QRadioButton" name="rbtnICDry">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>83</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>DRY</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnICRst">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>83</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>RST</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnICAuto">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>83</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>AUTO</string>
      </property>
     </widget>
     <widget class="QRadioButton" name="rbtnICZonal">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>20</y>
        <width>83</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ZONAL</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="rstFileEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>70</y>
       <width>221</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addRstFileBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>70</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QLabel" name="initLabel">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>30</y>
       <width>241</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>TextLabel</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="InitCondTable">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>110</y>
       <width>571</width>
       <height>131</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>9</pointsize>
      </font>
     </property>
    </widget>
    <widget class="QGroupBox" name="groupBox_7">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>301</width>
       <height>51</height>
      </rect>
     </property>
     <property name="title">
      <string>Output Interval</string>
     </property>
     <widget class="QRadioButton" name="fixIntvBtn">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>111</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Fixed Interval</string>
      </property>
      <property name="checked">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QRadioButton" name="asigOutBtn">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>20</y>
        <width>161</width>
        <height>19</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Assign Output Time</string>
      </property>
     </widget>
    </widget>
    <widget class="QLineEdit" name="outIntvEdit">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>260</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addIntvBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>261</y>
       <width>51</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Add</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="OutIntvTable">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>310</y>
       <width>571</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="deleteIntvBtn">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>261</y>
       <width>51</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Delete</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabFlowBCSetting">
    <attribute name="title">
     <string>Flow BC Setting</string>
    </attribute>
    <widget class="QTableWidget" name="boundaryTable">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>601</width>
       <height>351</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>-410</x>
       <y>30</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>PushButton</string>
     </property>
    </widget>
    <widget class="QPushButton" name="inputFileBtn">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>111</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>使用檔案輸入</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>181</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>10</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Set up boundary condition</string>
     </property>
    </widget>
    <widget class="QPushButton" name="explainBoundaryBtn">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>61</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>說明</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tabMobileSetting">
    <attribute name="title">
     <string>Mobile Setting</string>
    </attribute>
   </widget>
   <widget class="QWidget" name="tabMobileBCSetting">
    <attribute name="title">
     <string>Mobile BC Setting</string>
    </attribute>
   </widget>
  </widget>
  <widget class="QPushButton" name="exportBtn">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>630</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>輸出SIF</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget_6">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>110</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>.2dm Mesh</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEditMeshFileName"/>
    </item>
    <item>
     <widget class="QPushButton" name="pbtnFileSelector">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_7">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>651</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6">
    <item>
     <widget class="QLabel" name="label_11">
      <property name="text">
       <string>.msh Mesh</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineMeshFilePath"/>
    </item>
    <item>
     <widget class="QPushButton" name="mshFileSelector">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="readMeshBtn">
      <property name="text">
       <string>讀入網格</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget_8">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>80</y>
     <width>551</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <widget class="QLabel" name="label_8">
      <property name="text">
       <string>Save Folder</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="saveFolderEdit"/>
    </item>
    <item>
     <widget class="QPushButton" name="saveFolderBtn">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QLabel" name="label_Complete">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>640</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
